server/knexfile.js

```diff
@@ -0,0 +1,50 @@
+// Update with your config settings.
+
+/**
+ * @type { Object.<string, import("knex").Knex.Config> }
+ */
+module.exports = {
+
+  development: {
+    client: 'pg',
+    connection: {
+      port: 5432,
+      user: "chrissy",
+      password: null,
+      database: "beam_api"
```

Comment:
Just a note to the team to remind y'all that you'll need to change the user/pw when migrating/seeing your local db